clients/src/main/java/org/apache/kafka/common/requests/ListConfigResourcesResponse.java

```diff
@@ -79,6 +79,7 @@ public Collection<ConfigResource> configResources() {
             ).collect(Collectors.toList());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
This seems wrong. Since the only use of this method is in `KafkaAdminClient.listClientMetricsResources`, which itself a deprecated method, I would just remove this method and put the code to create the `ClientMetricsResourceListing` instances directly in `KafkaAdminClient`.